lib/airbrake/rake/tasks.rb

```diff
@@ -100,7 +100,10 @@
 
     exit!(1) if [id, key].any?(&:nil?)
 
+    env = heroku_env['RAILS_ENV'] || ENV['RAILS_ENV']
```

Comment:
Sounds better. If it causes confusion, we could output a warning message when we don't know the heroku env, explaining we picked a default and why the value should match. This would allow them to edit the deploy hook if the default was inappropriate.

Just curious, does a dynamic URL value for the deploy hook? E.G "some-url?#{ENV['RAILS_ENV']}"?